site/en/docs/extensions/mv3/screen_capture/index.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: "layouts/doc-post.njk"
+title: "Audio recording and screen capture"
+seoTitle: "Chrome Extensions: Audio recording and screen capture"
+date: 2023-04-14
+updated: 2023-04-14
+description: How to record audio or video from a tab, window or screen
+---
+
+This guide explains different approaches for recording audio and video from a tab, window or
+screen using APIs such as [chrome.tabCapture][tabcapture] or [getDisplayMedia][get-display-media].
+
+## Common Use Cases
+
+### Screen Recording {: #screen-recording}
+
+For screen recording, use the [getDisplayMedia][get-display-media] API. This provides the user with
+the ability to select which tab, window or screen they wish to share and provides a clear indication
+that recording is taking place.
+
+{% Img src="image/wVNVUJS8Z8O04i1tJKSdsp6nkRQ2/Q78okJG9DlfNwzyf2mly.png", alt="Screen share dialog for example.com", width="728", height="610" %}
+
+```js
+// User will be prompted to select what they would like to share.
+const stream = await navigator.mediaDevices.getDisplayMedia({ audio: true, video: true });
+```
+
+If called within a content script, recording will automatically end when the user navigates to a new
+page. To record in the background and across navigations, use an
+[offscreen document][offscreen-documents] with the `DISPLAY_MEDIA` reason.
+
+### Tab capture based on user gesture
+
+In some use cases, the user invokes your extension for a specific tab by clicking on the action
+icon. In these cases, it is not desirable to show a secondary dialog asking the user what they would
+like to share.
```

Comment:
>In these cases, it is not desirable to show a secondary dialog box asking the user what they would like to share

Should this actually say, "You don't need a second dialog box to ask users what they want to share because this is covered by the built-in 'Chose  what to share' dialog box?

Reply:
That wasn't the intent actually. What I'm trying to say is that sometimes you want neither, which tabCapture gives you (it just picks the current tab and doesn't show any UI).

Reply:
Then I think you need to elaborate on this point and explain it. I can't regurgitate the point based on what you just said. It needs a deeper explanation.

Reply:
I've tried rewriting this. Let me know what you think.